src/ray/common/constants.h

```diff
@@ -64,6 +64,8 @@ constexpr char kRayVersion[] = "3.0.0.dev0";
 /// Name for cloud instance id env
 constexpr char kNodeCloudInstanceIdEnv[] = "RAY_CLOUD_INSTANCE_ID";
 
+constexpr char kPlacementGroupAntiAffinityLabelName[] = "_PG";
```

Comment:
also with the latest discussion with hangmin, we need to append `_PG_{id}` since a node could have multiple PGs

Reply:
I feel we don't need to do that - it's probably autoscaler's job to create the appropriate label from a constraint when provisioning the nodes.